<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HackRF_Controller</class>
 <widget class="QWidget" name="HackRF_Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Частота дискретизации</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_sampleRate">
     <property name="currentIndex">
      <number>10</number>
     </property>
     <item>
      <property name="text">
       <string>20 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 МГц</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 Мгц</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>МШУ, дБ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="lna_gain">
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="singleStep">
      <number>8</number>
     </property>
     <property name="pageStep">
      <number>8</number>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Регулировка усиления (VGA), дБ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="vga_gain">
     <property name="maximum">
      <number>62</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>38</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_amp_gain">
     <property name="text">
      <string>Amp (0/14 дБ)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
